README.md

```diff
@@ -0,0 +1,36 @@
+# Archipelago Protocol
+
+Archipelago is a protocol for creating permissionless marketplaces for buying
+and selling collections of NFTs that are priced in ETH.
+
+The key features of Archipelago markets are:
+
+- Permissionless, neutral, and decentralized at the protocol level
+- Allows exchanging NFTs for ETH or WETH
+- Off-chain bids and asks, with on-chain execution
+- Optional on-chain order approval, for smart contract participation
+- Trait bidding (e.g. a bid on all fuzzy squiggles) and
+  collection-wide floor bids.
```

Comment:
This is kind of *the* key feature, right?—maybe worth moving earlier?
E.g., "decentralized at the protocol level" has precedent; and "ETH or
WETH" is nice QOL, but is not transformative like floor bids are.

Similarly, maybe worth mentioning the user-facing benefits of the nonce
and cancellation semantics: cancelling all same-nonce asks/bids for a
token at once; linked 1-of-*n* (or *k*-of-*n*) orders?

Reply:
Updated. I moved the collection-wide and trait-wide bids higher in the feature set, but still after the most basic features, since trait-wide bids necessarily exist in the context of buying and selling NFTs.